packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -0,0 +1,195 @@
+import { Component, EventEmitter, Inject, Input, OnChanges, OnInit, Output, SimpleChanges } from "@angular/core";
+import { TranslateService } from "@ngx-translate/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+/**
+ * Component to display dropdown list based on the options passed as parameters. The dropdown component is based
+ * on the Angular Material MatSelect.
+ */
+@Component({
+	selector: "stark-dropdown",
+	templateUrl: "./dropdown.component.html"
+})
+export class StarkDropdownComponent implements OnInit, OnChanges {
+	@Input()
+	public defaultBlank?: boolean;
+	
+	/**
+	 *  dropdownId HTML "id" attribute of the element.
```

Comment:
There is no need to repeat the name of the property in the JSDocs, they are implicit in the Typescript code and compodoc will put it in the docs ;)

The same for the other properties in this class